demo/ThemeTransform.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang=en>
+
+  <head>
+    <meta charset=utf-8>
+    <title>ThemeTransform</title>
+    <style>
+      :root {
+        --control-panel-theme-visibility: visible;
+      }
+      body{
+        padding: 20px 20px 20px 300px !important;
+      }
+      .content_block {
+        padding-bottom: 50px;
+      }
+      #demoMenu a {
+        display: block;
+        color: black;
+      }
+      #demoTools {
+        background: #ccc;
+        padding: 15px;
+        position: fixed;
+        left: 20px;
+        top: 20px;
+        z-index: 10000;
+        width: 230px;
+        border-radius: 8px;
+      }
+      .demoArticleTitle {
+        color: red;
+        font-size: 2.0em;
+      }
+    </style>
+ 
+    <link href="https://en.wikipedia.org/w/load.php?debug=false&lang=en&only=styles&skin=vector&modules=skins.minerva.base.reset|skins.minerva.content.styles|ext.math.styles|" rel="stylesheet" type="text/css"></link>
+
+    <script src=build/wikimedia-page-library-transform.js></script>
+    <link rel=import href=ControlPanel.html>
+  </head>
+
+  <body>
+    <div id='demoTools'>
+      <control-panel></control-panel>
+      <div id='demoMenu'></div>
+    </div>
+    <div class='content' id='content'></div>
+    
+    <script>
+        
+(function() {
+
+  // The json files below were saved using the mobileview parameters in the URL below.
+  // eslint-disable-next-line max-len
+  // https://en.wikipedia.org/w/api.php?action=mobileview&format=json&noheadings=true&page=Playwright&pilicense=any&prop=sections%7Ctext%7Clastmodified%7Clastmodifiedby%7Clanguagecount%7Cid%7Cprotection%7Ceditable%7Cdisplaytitle%7Cthumb%7Cdescription%7Cimage%7Crevision%7Cnamespace&sectionprop=toclevel%7Cline%7Canchor%7Clevel%7Cnumber%7Cfromtitle%7Cindex&sections=all&thumbwidth=640
+
+  const fileNames = [
+    'en.Barack_Obama.json',
+    'en.Audioslave.json',
+    'en.Chris_Cornell.json',
+    'en.Gantz.json',
+    'en.Mycenaean_Greek.json',
+    'en.Kill_\'Em_All.json',
+    'en.Manchester_United_F.C..json',
+    'en.John_McCain.json',
+    'en.Playwright.json',
+    'en.Toast.json'
+  ]
+
+  const urls = fileNames.map(filename => `/DemoArticles/${filename}`)
+
+  const demoMenu = document.getElementById('demoMenu')
```

Comment:
Consider using a drop down here to match all the demos and providing both Parsoid and MediaWiki response types since Android uses Parsoid (via the Content Service) whenever possible.